src/components/Prose.astro

```diff
@@ -1,5 +1,5 @@
 <article
-  class="prose prose-app my-12 max-w-none text-base prose-strong:text-white prose-table:text-base prose-a:text-app-purple hover:prose-a:text-app-pink prose-headings:text-app-green text-white"
+  class="prose prose-app my-12 max-w-none text-base prose-table:text-base prose-a:text-app-green hover:prose-a:text-app-pink text-white"
```

Comment:
This change broke the text on the member pages, maybe there's a quick fix here?

![2024-08-05-19-37-03](https://github.com/user-attachments/assets/230d7089-639d-4c8f-85c6-5fb2fbb6513c)

Reply:
ah, yeah, what you said makes sense.